src/components/Auth/SignIn.js

```diff
@@ -55,7 +55,7 @@ export default class SignIn extends Component {
 
   renderSignUp(userAddress, toggleModal) {
     return (
-      <FormDiv>
+      <Form onSubmit={e => e.preventDefault()}>
```

Comment:
Let's create a `Form` component which internally does the `preventDefault` and re-use that instead of rewriting this code everywhere.